RoaringBitmap/src/main/java/org/roaringbitmap/longlong/LongUtils.java

```diff
@@ -125,4 +125,43 @@ private static int compareTo(byte[] buffer1, int offset1, int length1,
   public static long initWithFirst4Byte(int v) {
     return ((long) v) << 32;
   }
+
+  /**
+   * shift the long left by the container size amount so we can loop across containers by +1 steps
+   * @param num long being treated as unsigned long
+   * @return value shifted out of value space into container high part
+   */
+  public static long leftShiftHighPart(long num) {
```

Comment:
Is the name right ? not `rightShiftHightPart` ?

Reply:
OMG, you are so correct. Thank you for spotting..